addons/mass_mailing_sms/static/tests/tours/mass_mailing_activities_split_test_tour.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('mass_mailing_activities_split_test_tour', {
+    test: true,
+    url: '/web',
+    steps: [
+        {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open Email Activities',
+            trigger: '.o-mail-ActivityGroup[data-model_name="mailing.mailing"] span:contains("Email Marketing")',
+        }, {
+            content: 'Open Email Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New Email!")',
+            run: () => {
+                const kanbanCards = document.querySelectorAll('.o_kanban_record .o_kanban_record_headings h3 span');
+                for (let i = 0; i < kanbanCards.length; ++i) {
+                    if (kanbanCards[i].innerText == "New SMS!") {
+                        console.error('SMS Marketing record should not appear in this view');
+                    }
+                }
+            },
+        }, {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open SMS Activities',
+            trigger: '.o-mail-ActivityGroup[data-model_name="mailing.mailing"] span:contains("SMS Marketing")',
+        }, {
+            content: 'Open SMS Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New SMS!")',
+            run: () => {
+                const kanbanCards = document.querySelectorAll('.o_kanban_record .o_kanban_record_headings h3 span');
+                for (let i = 0; i < kanbanCards.length; ++i) {
+                    if (kanbanCards[i].innerText == "New Email!") {
+                        console.error('SMS Marketing record should not appear in this view');
+                    }
+                }
```

Comment:
Why not just check that `$(".o_kanban_record:contains(New Email!)").length === 0`?
`.o_kanban_record_headings h3 span` are all leaking a implementation details that are irrelevant for the test.

Also I think the `console.error` string refers to "SMS Marketing records not appearing in this view". I think you meant Email Marketing records not appearing there (or _only_ SMS Marketing records appearing there).